vector/src/main/java/im/vector/app/core/ui/list/GenericButtonItem.kt

```diff
@@ -70,6 +75,12 @@ abstract class GenericButtonItem : VectorEpoxyModel<GenericButtonItem.Holder>()
         val textStyle = if (bold) Typeface.BOLD else Typeface.NORMAL
         holder.button.setTypeface(null, textStyle)
 
+        holder.button.rippleColor = if (highlight) {
```

Comment:
was `R.attr.colorSecondary` also the previous highlight colour? just in case we've changed the default value for a generic component which might be used in other places

Reply:
Good point, button has `style="@style/Widget.Vector.Button.Text"` and this style has `<item name="colorControlHighlight">?colorSecondary</item>`. But I will check if `rippleCount` has the same effect with `colorControlHighlight`.

Reply:
thanks!

Reply:
Actually was not the exact color. Sorry, now fixed.